modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -268,6 +271,10 @@ public SpringCodegen() {
             generatedConstructorWithRequiredArgs));
         cliOptions.add(new CliOption(RESOURCE_FOLDER, RESOURCE_FOLDER_DESC).defaultValue(this.getResourceFolder()));
 
+        cliOptions.add(CliOption.newString(USE_HTTP_SERVICE_PROXY_FACTORY_INTERFACES_CONFIGURATOR,
+            "Generate HttpInterfacesAbstractConfigurator based on an HttpServiceProxyFactory instance (as opposed to a WebClient instance, when disabled) for generating Spring HTTP interfaces. Requires spring-web 6.1+.")
```

Comment:
>Requires spring-web 6.1+.

Is this still true after removing the two imports mentioned in my other comment?

Reply:
The mention of spring-web 6.1+ is still there.